siliconcompiler/schema/schema_cfg.py

```diff
@@ -4,7 +4,13 @@
 import json
 import re
 
-from .utils import trim
+# Default import must be relative, to facilitate KLayout's direct import of the
```

Comment:
While klayout is the tool at the moment with this issue, others will come. Maybe change the comment to reflect that it's for the tools with python interfaces (like klayout) that need this.